includes/definitions/discovery/zynos.yaml

```diff
@@ -26,6 +26,61 @@ modules:
                 oid: sysMgmtCPU5SecUsage
                 num_oid: '.1.3.6.1.4.1.890.1.15.3.2.7.{{ $index }}'
     sensors:
+        pre-cache:
+            data:
+                -
+                    oid:
+                        - ifDescr
```

Comment:
ifDescr is not necessary here, unused.

Reply:
Well spotted, I originally used ifDescr but must have forgotten to remove it from the pre-cache